content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Using Electronic Health Records to Develop Real-Time COVID-19 Indicators"
+author: "Aaron Rumack"
+date: 2020-10-14
+tags: ["nowcasting", "data sources"]
+output:
+  html_document:
+    code_folding: hide
+---
+
+In addition to symptom surveys, one of our most prized data sources comes from health insurance claims data. Healthcare partners provide us with the counts of people going to the doctor due to COVID-like illness in counties across the entire country. We use this data to calculate a signal that estimates the level of COVID-like illness in every county where enough data is available. In this post, we discuss the data itself, what this data can tell us that the number of confirmed cases can't, and how we calculate a %CLI signal. We also investigate why the data appeared to become much less useful over time, and how we discovered that the data just needed a simple adjustment!
+
+<!--more-->
+
+In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from this data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness.
```

Comment:
Do we want to discuss the hospital records?  (1) we were not supposed to receive them based on our DUA; and (2) we no longer receive them.  The alternative is to discuss the insurance claims only.  If we want to show some results that involve the hospital records, we can add a footnote that these results are based on both insurance claims and hospital records (to which we had access until October 1).
If we decide to continue to discuss hospital records, note that they do NOT cover 10%-15% of the US population, but rather only about 40% of that 10%-15%, namely 4%-6%.